lib/src/wolt_modal_sheet.dart

```diff
@@ -332,8 +331,17 @@ class WoltModalSheetState extends State<WoltModalSheet> {
     Widget modalContent = ValueListenableBuilder(
       valueListenable: widget.pageIndexNotifier,
       builder: (context, currentPageIndex, __) {
+        if (_pages.isEmpty) {
```

Comment:
maybe we can wrap to try-catch for prevent the app from crashing if there’s an issue with pageListBuilderNotifier


```suggestion
  if (_pages.isEmpty) {
    try {
      final initialPages = widget.pageListBuilderNotifier.value(context);
      assert(
        initialPages.isNotEmpty,
        'pageListBuilder must return a non-empty list.',
      );
      _pages = initialPages;
    } catch (e) {
      debugPrint("Error initializing pages: $e");
    }
  }
```

Reply:
Thanks for the input. I would like to keep it the current way because it should throw exception if the initial list is empty. We also provide assert which works in debug mode for feedback to devs.